src/phases/switch-summon-phase.ts

```diff
@@ -65,11 +65,11 @@ export class SwitchSummonPhase extends SummonPhase {
 
     if (!this.doReturn || (this.slotIndex !== -1 && !this.getAlliedParty()[this.slotIndex])) {
       if (this.isPlayer) {
-        return this.switchAndSummon();
-      } else {
-        time.delayedCall(750, () => this.switchAndSummon());
+        this.switchAndSummon();
         return;
       }
+      time.delayedCall(750, () => this.switchAndSummon());
+      return;
```

Comment:
```suggestion
      } else {
        time.delayedCall(750, () => this.switchAndSummon());
      }
      return;
```

now that is confusing. I think this might be clearer?

Reply:
Yeah I could just pull the `return` out of the `if` block entirely
```ts
      if (this.isPlayer) {
        this.switchAndSummon();
      } else {
        time.delayedCall(750, () => this.switchAndSummon());
      }
      return;
```